### DISCUSSION TITLE
Solarman Logger Serial 27XX

### DISCUSSION THREAD

**Discussion by marco-dea** [StephanJoubert/home_assistant_solarman#313] [General]
Hi Community.
I'm glad to join this group. I'm an IT Project manager and I started to use Home Assistan to monitor my home devices. Recently I've istalled the Solarman Intergration to monitor my Solar device (logger serial number 23xx) and it's amazing the effortless and detailed info available.

I'd provide the same monitoring system to a friend of mine with a similar Solare equipment. We have the same devices but his logger serial number is 27xx. Anyway I haven't manage to setup the integration.

Are the serial number 27xx supported by this integration? 

Kind regards

**Comment by webartoli** [comment]
I've the same issue with a device like this: 

Inverter:
- Brand: Zucchetti ZCS
- Model No: AZZURRO 1PH HYD3600 ZSS HP

Logger:
- S/N: 27xxxx
- Firmware: LSW3_15_270A_1.53

Trying `sofar_hyd3k-6k-es.yaml` the results is **Unknown** on all sensors.
Trying `zcs_azzurro-ktl-v3.yaml` the results that some sensor are running and seems not the right inverter definition.

**Reply by webartoli** [reply]
Solved using `hyd-zss-hp-3k-6k.yaml` definition that is on main branch but currently is not yet released.
Adding definition manually it work like a charm.

**Reply by RazorCopter** [reply]
AFORE Inverter 

Embedded_Device_SN: 270173xxxx
Module_Version_No: LSW3_15_1E20_1.14

I have this, can you tell me how you got it to work?

**Comment by RazorCopter** [comment]
Marco hai risolto? in questo momento sto usando ilc cloud ma mi piacerebbe integrarlo in locale
